lib/counter_culture/counter.rb

```diff
@@ -290,7 +290,7 @@ def previous_model(obj)
       changes_method = ACTIVE_RECORD_VERSION >= Gem::Version.new("5.1.0") ? :saved_changes : :changed_attributes
       obj.public_send(changes_method).each do |key, value|
         old_value = ACTIVE_RECORD_VERSION >= Gem::Version.new("5.1.0") ? value.first : value
-        prev.public_send("#{key}=", old_value)
+        prev[key] = old_value
```

Comment:
Reading up the API documentation, honestly this looks to be the "more correct" way to do this anyhow. So if it also fixes your issue with aasm, all the better!